### PR TITLE
honeycomb logging

### PR DESCRIPTION
**Pull request by boris-dimitrov**
Honeycomb is a super basic analytics DB that lets you filter and group timestamped log events and chart sum/avg/percentile of any event field over time.

The rails code here emits stats that are already being gathered by rails.  For example, db_runtime, the total latency of DB queries in the request.  We can extend this event listener pattern as needed to emit more information.   The idea is to keep all logging here.

To use this, you need to set IDSEQ_HONEYCOMB_WRITE_KEY to a special value and IDSEQ_HONEYCOMB_DATA_SET to idseq_web.   Ryan can invite you to our team account on honeycomb so you can issue queries against the events.   Events show up pretty instantaneously.

TODO:  Add event fields for environment, host,  logged in user id, more per-request stats.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -27,6 +27,10 @@ services:
       - redis
     environment:
       - SAMPLES_BUCKET_NAME=czbiohub-idseq-samples-development
+      - IDSEQ_AIRBRAKE_PROJECT_ID
```
Comment (boris-dimitrov) [inline docker-compose.yml]: We can make some changes to idseq_infra and airbrake.rb to take advantage of this.

File: config/initializers/honeycomb.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
+require 'libhoney'
```
Comment (boris-dimitrov) [inline config/initializers/honeycomb.rb]: The code in this file was dumped verbatim from the "getting started" example.

Comment (ryanking) [reply config/initializers/honeycomb.rb]: maybe drop a link to that example in a comment here?

Comment (boris-dimitrov) [reply config/initializers/honeycomb.rb]: Great idea.   I also found every logging/profiling tool ever made for Rails by googling for this line from the example:
```
 ActiveSupport::Notifications.subscribe(/process_action.action_controller/) do
```
They all use it :)

**Pull request merged**